lib/message.rb

```diff
@@ -0,0 +1,11 @@
+class Message
+  attr_reader :text, :translator
+  def initialize(message, morse_translator)
+    @text = text
+    @translator = translator
+  end
+
+  def translation
+
```

Comment:
It looks like you are on the right track here, but this method seems to be... missing... something